examples/open-records/open_records.bal

```diff
@@ -11,7 +11,6 @@ type Employee record {
     int id;
 };
 
-// Adds an additional `id` field to `e`.
```

Comment:
Should we add a similar comment before L21 instead for `country`?

Reply:
Yes better. Added in 0025d0c0c85d7f818fbf4d2afc086cbf2917e5a1